client/src/main/kotlin/no/nav/su/se/bakover/client/krr/KontaktOgReservasjonsregisterClient.kt

```diff
@@ -25,16 +27,31 @@ class KontaktOgReservasjonsregisterClient(
     private val log = LoggerFactory.getLogger(this::class.java)
 
     override fun hentKontaktinformasjon(fnr: Fnr): Either<KontaktOgReservasjonsregister.KunneIkkeHenteKontaktinformasjon, Kontaktinformasjon> {
-        val (_, response, result) = "${config.url}$PERSON_PATH".httpGet()
+        val request = serialize(
+            HentKontaktinformasjonRequest(
+                personidenter = listOf(fnr.toString()),
+            ),
+        )
+        val (_, response, result) = "${config.url}$PERSONER_PATH".httpPost()
             .authentication().bearer(azure.getSystemToken(config.appId))
             .header("Accept", "application/json")
+            .header("Content-Type", "application/json")
```

Comment:
Kan bruke `HttpHeaders` for alle headers som er default her.  også ContentType.Applicaiton.json.toString() feks